sass/themes/schemas/components/light/_list.scss

```diff
@@ -237,9 +275,9 @@ $fluent-list: extend(
 /// @prop {Map} header-background [contrast-color: ('gray', 900)] - The list header background color.
 /// @prop {Map} item-background [contrast-color: ('gray', 900)] - The list item background color.
 /// @prop {Map} item-text-color [color: ('gray', 900)] - The list item text color.
-/// @prop {Map} item-text-color-hover [color: ('gray', 900)] - The list item text color.
+/// @prop {Map} item-text-color-hover [color: ('gray', 900)] - The list item text color on hover.
 /// @prop {Map} item-text-color-active [color: ('gray', 900)] - The active list item text color.
```

Comment:
You should also update the item-text-color-selected here.

Reply:
@SisIvanova Done.